packages/paste-core/components/form-pill-group/src/FormPillGroup.tsx

```diff
@@ -49,6 +49,13 @@ export interface FormPillGroupProps
    * @memberof FormPillGroupProps
    */
   size?: FormPillGroupSizeVariant;
+  /**
+   * The variant of the FormPillGroup to use. The 'tree' option allows for more data to be displayed on select and still allows for select states.
```

Comment:
suggestion: add some info about this changing the underlying aria role just to be more transparent for users. and what the default one is

Reply:
Updated in latest commit